webpack.dev.js

```diff
@@ -0,0 +1,28 @@
+const merge = require('webpack-merge');
+const config = require('./webpack.config.js');
+const WriteFilePlugin = require('write-file-webpack-plugin');
+
+module.exports = merge(config, {
+  mode: 'development',
+  devtool: 'source-map',
+  devServer: {
+    proxy: {
+      '/extensions': {
+        target: 'http://localhost:3001',
+        pathRewrite: { '^/extensions': '/public/extensions' }
+      }
+    },
+    port: 3001
+  },
+  plugins: [
+    ...config.plugins,
```

Comment:
I believe using webpack-merge means you don't have to array-spread your existing config

Reply:
You're right :+1: